auslib/web/admin/views/required_signoffs.py

```diff
@@ -74,6 +90,43 @@ def get(self, input_dict):
 
         return jsonify(count=total_count, required_signoffs=revisions)
 
+    def get_all(self):
```

Comment:
Same question here...what does this do that `get()` doesn't?

Reply:
Same thing with the case above.